build.gradle

```diff
@@ -70,6 +68,14 @@ processResources {
   }
 }
 
+configurations {
+    provided
+}
+
+sourceSets {
+    main { compileClasspath += configurations.provided }
+}
+
```

Comment:
Why is this required? Any docs mentioning this? Looks pretty standard.

Reply:
It came from here and replaces `org.springframework.build.gradle:propdeps-plugin` (which I had problems with).

Reply:
Still working through the updates at work... but noticed the gradle 7 support in Springboot 2.5.0.

Reply:
The code above is not needed. The whole provided scope is a maven thing. We can just use implemenation for the install4j dependency.
Basically we solved all issues here except for 1 issue with install4j that we can't fix ourselves on building a release.


https://github.com/FAForever/downlords-faf-client/commits/feature/gradle7

Maybe updating to install4j 9 can fix it. Otherwise we need to raise a bugticket.